hyperjumptech/monika
# Fix: Cluttered report data from monika to symon

# Monika Pull Request (PR)

## What feature/issue does this PR add
#1197 Cluttered Report data

## How did you implement / how did you fix it
1. Remove cluttered unused data to send probe report in symon-mode

## How to test
1. run symon
2. run monika in symon-mode
3. make sure the new report is entering to the symon report

![Screenshot 2023-11-27 at 18 38 45](https://github.com/hyperjumptech/monika/assets/8952493/11ad4a0d-05cc-4b71-812b-8ec07d5790a6)


![Screenshot 2023-11-27 at 18 38 04](https://github.com/hyperjumptech/monika/assets/8952493/0b5aed15-9b7c-4c9f-9b68-578234f56663)
